popup.js

```diff
@@ -0,0 +1,198 @@
+  details: `Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s. Lorem Ipsum is simply dummy text of the printing and typesetting industry. 
+                Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s.`,
+},
+
+{
+  id: 4,
+  name: 'Multi-Post Stories',
+  img: './images/desktop-img/Snapshoot Portfolio(4).png',
+  type: 'Canopy',
+  dev: 'Back End Dev',
+  year: 2015,
+  source: 'link',
+  sourceIcon: 'img-link',
+  liveLink: 'link to life project',
+  stack: {
+    1: 'HTML', 2: 'CSS', 3: 'JavaScript', 4: 'Ruby', 5: 'Bootstrap',
+  },
+  tag: {
+    1: 'pattern', 2: 'poster', 3: 'abstraction', 4: 'people', 5: 'technology', 6: 'media', 7: 'nature', 8: 'food',
+  },
+  price: '45',
+  btn: 'Choose illustration',
+  title: 'Nature',
+  tagline: 'catchy look inspiring bionic form',
+  details: `Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s. Lorem Ipsum is simply dummy text of the printing and typesetting industry. 
+                Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s.`,
+}];
+
+const displayModal = (element) => {
+  modalWrapper.innerHTML = `
+    <div class="ven-modal ven-overlay">
+        <ul class="ven-modal-ul">
+            <li class="ven-modal-ul-li">
+                <ul class="ven-title-canopy-cancel-wrapper">
+                    <li class="ven-title-canopy-year">
+                        <h2>${element.name}</h2>
+                        <ul class="ven-canopy-list ven-canopy-list-wrapper">
+                            <li><b>${element.type}</b></li>
+                            <li class="ven-card-empty-list"></li>
+                            <li>${element.dev}</li>
+                            <li class="ven-card-empty-list"></li>
+                            <li>${element.year}</li>
+                        </ul>
+                    </li>
+                    <li>
+                        <img src="./images/icons/Icon.svg" alt="close icon" class="ven-cancel-icon" style="cursor: pointer">
+                    </li>
+                </ul>
```

Comment:
- [ ] Kindly style the popup well the technology and the button
- Your Design
<img width="1320" alt="Screenshot 2023-05-18 at 08 47 58" src="https://github.com/Shabiblac/Microverse-Portfolio-Design/assets/81444338/bc47a556-24f6-49fc-9a91-09459fab2122">


- Figma Design
<img width="1433" alt="Screenshot 2023-05-18 at 08 43 05" src="https://github.com/Shabiblac/Microverse-Portfolio-Design/assets/81444338/746800f0-899c-4720-84dd-331532132719">